sql/updates/world/3.3.5/9999_99_99_99_world_335.sql

```diff
@@ -0,0 +1,24 @@
+-- Coosh'coosh
+UPDATE `creature_template` SET `AIName` = 'SmartAI', `ScriptName` = '' WHERE `entry` = 18586;
+
+DELETE FROM `smart_scripts` WHERE (source_type = 0 AND entryorguid = 18586);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(18586, 0, 0, 0, 0, 0, 100, 0, 2000, 2000, 5000, 5000, 11, 9532, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Coosh\'coosh - In Combat - Cast \'Lightning Bolt\''),
```

Comment:
Changes the cast to CMC with 3 secs repeat.
2000, 2000, 3000, 3000, 11, 9532, 64

Reply:
cmc? sorry, unfamiliar with the acronym